tests/test_actuators.py

```diff
@@ -68,7 +68,7 @@ def test_unidirectional(self):
         a = actuators.UnidirectionalActuator(direction=None)
         self.assertEqual(a.next_move(), constants.RIGHT)
 
-    def test_pathfinder(self):
+    def test_pathfinder_bfs(self):
```

Comment:
We're missing the equivalent `test_pathfinder_astar()`.

Reply:
There is test_pathfinder_astar below this method.